File: specification/mediaservices/resource-manager/Microsoft.Media/Encoding/stable/2022-07-01/examples-js/transforms-get-by-name.js
const { AzureMediaServices } = require("@azure/arm-mediaservices");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets a Transform.
 *
 * @summary Gets a Transform.
 * x-ms-original-file: specification/mediaservices/resource-manager/Microsoft.Media/Encoding/stable/2022-07-01/examples/transforms-get-by-name.json
 */
async function getATransformByName() {
  const subscriptionId =
    process.env["MEDIASERVICES_SUBSCRIPTION_ID"] || "00000000-0000-0000-0000-000000000000";
  const resourceGroupName = process.env["MEDIASERVICES_RESOURCE_GROUP"] || "contosoresources";
  const accountName = "contosomedia";
  const transformName = "sampleTransform";
  const credential = new DefaultAzureCredential();
  const client = new AzureMediaServices(credential, subscriptionId);
  const result = await client.transforms.get(resourceGroupName, accountName, transformName);
  console.log(result);
}

File: specification/mediaservices/resource-manager/Microsoft.Media/Encoding/stable/2022-07-01/examples-java/transforms-get-by-name.java
/** Samples for Transforms Get. */
public final class Main {
    /*
     * x-ms-original-file: specification/mediaservices/resource-manager/Microsoft.Media/Encoding/stable/2022-07-01/examples/transforms-get-by-name.json
     */
    /**
     * Sample code: Get a Transform by name.
     *
     * @param manager Entry point to MediaServicesManager.
     */
    public static void getATransformByName(com.azure.resourcemanager.mediaservices.MediaServicesManager manager) {
        manager
            .transforms()
            .getWithResponse("contosoresources", "contosomedia", "sampleTransform", com.azure.core.util.Context.NONE);
    }
}
